.github/workflows/drizzle_ci.yml

```diff
@@ -0,0 +1,81 @@
+name: Drizzle Package Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I think we want it to run on all pull requests, not just those against master, and we also want it to run when we push tags.  So something like:

```yaml
on:
  push:
    branches:
      - master
    tags:
      - "*"
  pull_request:
```

Reply:
I made these changes.